src/neo-vm/ExecutionEngine.cs

```diff
@@ -277,14 +277,18 @@ private bool ExecuteInstruction()
                                 context.InstructionPointer += 3;
                             return true;
                         }
-                    case OpCode.CALL:
+                    case OpCode.CALL_I:
```

Comment:
Maybe we can rename `CALL_I` to `CALL` ?

Reply:
For me, the logic change by default to be isolated is good, but i think that we can call it as `CALL`

Reply:
I am wondering if we really need stack isolation in the case of executing internal calls.

Reply:
Agree with you, but then we should remove `CALL_I` not `CALL`

Reply:
Yes. I will confirm with @lightszero.